inscriptions/ISic003683.xml

```diff
@@ -192,32 +197,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140928">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>Λ</abbr>
-                        <ex>ούκιος</ex>
-                    </expan>
-                    ∙ Κα<supplied reason="lost">στρί</supplied>κιος ∙
-                    <expan>
-                        <abbr>Κυρ</abbr>
-                        <ex>είνα</ex>
-                    </expan>
-                    ∙ Προύδηνς ∙ ἱππεὺς ∙
-                    <expan>
-                        <abbr>Ῥωμ</abbr>
-                        <ex>αίων</ex>
-                    </expan>
-                    ∙ πρῶτος ∙ Μελιταίων<lb n="2"/>καὶ ∙ πάτρων ∙ ἄρξας ∙ καὶ ∙ ἀμφιπολεύσας ∙ θεῷ ∙ Αὐγούστῳ
-                    <lb n="3"/>
-                    <supplied reason="lost">— — — — — —</supplied>ΕΣΧ<supplied reason="lost">— —</supplied>Ν<supplied reason="lost">— —</supplied>Ε․Ι․ΝΕ
-                    <supplied reason="lost">— — — — — — — —</supplied>
-                    <lb n="4"/>
+                   <lb n="1"/><persName type="attested"><name type="praenomen"><expan><abbr>Λ</abbr><ex>ούκιος</ex></expan></name> <g type="interpunct"/> <name type="nomen">Κα<supplied reason="lost">στρί</supplied>κιος</name> <g type="interpunct"/> <orgName type="tribus"><expan><abbr>Κυρ</abbr><ex>είνα</ex></expan></orgName> <g type="interpunct"/> <name type="cognomen">Προύδηνς</name></persName> <g type="interpunct"/> ἱππεὺς <g type="interpunct"/> <placeName type="ethnic"><expan><abbr>Ῥωμ</abbr><ex>αίων</ex></expan></placeName> <g type="interpunct"/>πρῶτος<g type="interpunct"/>Μελιταίων
```

Comment:
spacing missing near the end; and change the g types to g refs with the interpuncts